damienharper/auditor-bundle/1.0/config/packages/dh_auditor.yaml

```diff
@@ -0,0 +1,5 @@
+dh_auditor:
+    # Configuration reference available at:
+    # http://localhost:8000/docs/auditor-bundle/configuration/reference.html
```

Comment:
Would an external url to the repot be better?

Reply:
Sure, it wasn't expected to be the local development URL